Dfe.Academies.Academisation.Data/AcademisationContext.cs

```diff
@@ -104,9 +105,23 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
 		OnAdvisoryBoardDecisionCreating(modelBuilder);
 
+		OnProjectCreating(modelBuilder);
+		
 		base.OnModelCreating(modelBuilder);
 	}
 
+	private static void OnProjectCreating(ModelBuilder modelBuilder)
+	{
+		modelBuilder.Entity<ProjectState>()
+			.Property(e => e.GoverningBodyResolution).HasConversion<string>();
+		modelBuilder.Entity<ProjectState>()
+			.Property(e => e.Consultation).HasConversion<string>();
+		modelBuilder.Entity<ProjectState>()
+			.Property(e => e.DiocesanConsent).HasConversion<string>();
+		modelBuilder.Entity<ProjectState>()
+			.Property(e => e.FoundationConsent).HasConversion<string>();
```

Comment:
Do we need to map the LegalRequirementsSectionComplete?

Reply:
No, entity framework knows how to map boolean's
The above change makes sure that enum values are stored in the db as strings. This follows on from the convention used already in this API.

Reply:
Ah I see!